### PR TITLE
Spacinghelpersmoreverbose

### PR DESCRIPTION
**Pull request by orthagonal**

### CODE REVIEW

File: tasks/css.js [hunk 1]
```diff
@@ -47,7 +47,8 @@ module.exports = function (config, base, outputName, input) {
   });
 
   Object.keys(config.spacing.default).forEach(spacing => {
-    cssVars[`spacing-${spacing}`] = config.spacing.default[spacing];
+    const sizeString = spacing === 'none' ? '0' : spacing;
+    cssVars[`spacing-${sizeString}`] = config.spacing.default[spacing];
```
Comment (jgallen23) [inline tasks/css.js]: you can leave this one as none

File: styles/mixins/spacing.js [hunk 2]
```diff
@@ -8,7 +8,8 @@ module.exports = function (config) {
 
     for (const breakpoint of spacingBreakpoints) {
       styles[breakpoint] = {};
-      styles[breakpoint][`${prop}-${position}`] = config.spacing[breakpoint][size];
+      const sizeKey = size === '0' ? 'none' : size;
```
Comment (jgallen23) [inline styles/mixins/spacing.js]: dont need this anymore, right?

Comment (jgallen23) [inline styles/mixins/spacing.js]: ping?

Comment (orthagonal) [inline styles/mixins/spacing.js]: When I leave out this check at line 11, it churns out an undefined node.  The output looks right if I leave it in.

Comment (jgallen23) [inline styles/mixins/spacing.js]: how does size ever equal 0 now? I thought we changed it

File: styles/mixins/spacing.js [hunk 3]
```diff
@@ -16,7 +16,8 @@ module.exports = function (config) {
 
   return function (mixin, prop, position, size) {
     if (prop && position && size) {
-      return spacingMixin(prop, position, size);
+      const sizeKey = size === '0' ? 'none' : size;
```
Comment (jgallen23) [inline styles/mixins/spacing.js]: what file is still passing 0?  we need to update that

Comment (orthagonal) [inline styles/mixins/spacing.js]: Tracking it down and nuking it now.

**Comment by jgallen23** [comment]
@dawnerd @chrisopedia can you guys sign off on this too?

File: conf/default-spacing.yaml [hunk 4]
```diff
@@ -1,6 +1,6 @@
 spacing:
   default:
-    0: 0
+    none: 0px
```
Comment (chrisopedia) [inline conf/default-spacing.yaml]: we should leave off the unit for a 0 value.

Comment (orthagonal) [inline conf/default-spacing.yaml]: Done.

**Comment by jgallen23** [comment]
closing because other PR had this functionality

**Pull request closed without merging**